src/logic.py

```diff
@@ -69,12 +72,14 @@ def action_dispatcher(ip: str, prev_ip: str, active_ip: str, passive_ip: str):
         if css_maintenance_to_active:
             logger.info("active_ip")
             dispatch_css_maintenance_action(False)
+            dispatch_uptime_incident(False)
         else:
             logger.info("Failed to turn off the css maintenance mode")
     elif (ip == passive_ip and prev_ip == active_ip):
         logger.info("passive_ip")
         dispatch_action_by_id(config.get('gh_workflow_id'))
         dispatch_css_maintenance_action(True)
+        dispatch_uptime_incident(True)
```

Comment:
@thegentlemanphysicist can we add some exception handling so that initial dispatch action do not interrupt `dispatch_uptime_incident`??